notebooks_community/vbll_thompson_sampling.ipynb

```diff
@@ -0,0 +1,803 @@
IrnxvYFkz2CyZbW1BOVTJ45JADE2DdSH0dgY2KaSHYfB5P3gnw3H7XbrNfibJyKi8JDgINVh00v+hd3t8emOhAQT7S6P9pmwmk26MyGloVImGqEb0TbSckleQkJHtZZCJjjPapvqlPNy7dcBWL2ZMzQBMjiSO1RmsxmzZt+gF/AEejua0fXu39B37iDKGhuxpLYR/zjYjFMtXTh95H3gyPsw2ROROm8l0j67Gl0pmdjamod/dS3C5BN/w1LP20g0B3I6DjX4sKHRh1XTrchLMePrM4APspJwxlKKKdNnYW75QpQvvBHTZlyn9xHk6k/ElOZcyY3v625Mas1OTPLvhzljNZAyC9EWUgDx5JNPap7ClRw/fhwzZ0p3ruiQnImf/vSnUft6RETxHExIpYZcBRmJ2u1S+0x0u9Ha5UGjU1ppQ4d7JdstkZvLYbbBnVmml+RLJLSf1p0Je1edNnTShk91e+BJKdLn9GZMH1UlhyM9B44vPqHTR2fX7sQX3E489KAX9f5MVPbk4mxtExobzussjo66f6PJUQxnXjmsKdk4v+B/saXvfnyhaysWmE5izmcSYEnMQEPqXBRl2DErxYlHUtPgS5oA5xRJsMy54vftyinvDyQk2fI9WLvaYdgvGU8eJSEdYVy4cAEtLZ+0+xxOSUmJVl1E6whjuB2I4uJiHmEQEUWR2ys7E3LU4dE+E3LsIWWPki8hOxPRyJswe7oCwUTbSQ0iBkizqv5gwp0xTaskRkqOTqQBVFJzhVaOSFmmjPSWltSDu05K3429py5g06E6nG8PlHiW2FrwP/ln8NmJfu29IcPGJClU2mDr0cRVJFgO4feio/4UZs0uj+8ciGASpSRQSiJl8BvJyclhEiUR0TgiHRJlAJTsTshRR1evVys69KjDbtUXz4gddfSTQVeSKyGtowdXcgwEExnTAsHECId7WXua9QjB1t0wMNZbGkldWqIpQdSBqha8dKgWVRe6YYYfCy2ncXdODWbnJSMlyQFX9hxY3G16HDM4wfJqjmBiMomypqYGra2t2LJlC37zm9/grbfe0senTZuGlJTAoshRhxxBfPWrX9X35XhkzZo1Q8o4jxw5wjJOIqJxnoQpAYUcdUgDKzn6QP9Rh+xORHrYl9ndAUf76f8OJvoHagWCiek60yIkhqEDtlLq98HkdWtw0jPxM+guuOm/Jm3KOlTUtOPlQ7U4Vu9EOrqw3HIYN2c5cV1BGtIyc+BJKdTdE+mBIcO2nJO/CI+09463AOKBBx7QQOBSu3fvxq233hr44iYTnnvuOX3u4EZSGzZs0B2LpUuXYv369SgrK7vqr8sAgojo2iUVHBJM6FFHV+CoQ3YspERUAgqHzRzRqo5AMHFGyylt3fUDI8eD7agDwURpSK20zX3d2o0yOAND5lp0Ft+qQclwn+N4g1MDiffOtqLMVIdl5g8wLT3Q4TK9oBTWvq6BGRlyNNJZdMtlB23FZAAxVhhAEBGND7K939V/1HGxy63VHa4+v04MlSRMCSgiOT1UcyY6Pta8CbtMCjX8n9xbQpp2kpTx430pRZ86l0LYnWeRWrMLFnfgyN6TPhVO6WSZMPzORvXFbvyzog7vnj6PG03HcIOpChNS7CgryEJhbg5s7sDEzit1sGQAEUYMIIiIxh95KXL17060dXvQ0u3Row6v349EmzXiiZhazdFRpTsTduc5naw5cG/WBLjTpmpAIfkJhvUKR+r+Pq2OSG4a1EgqbxF6cudftg13Y0cvNlfW4ejx47gFFcgxtSHNYUNZQTZKMm2BIEoTLJegJ2f+kF0NBhBhxACCiGj8kyoGCSY6Bydien2wmM0DiZgR253w9+lcDt2d6KiCua/nkz8zmTVXwZ0hwcRU+ByZl+2gKUmW9s7aQePCl2sfi8uRwWavHa5DzYf7UO47CrupDyk2E2bmJGFqThqsVlv/iPAvDSR/MoAIIwYQRESxRV6mpIGVBBOyO9GqiZhe+P3QQCLZboVd+j5H5Iv7tdLCLrsTHVWwuoa2MvA5MvWYw50+FX3JhUOPOgxD+1OknN87EIRI6+2uoluuWEra4/Fi1wcfo+3INhT0nZNPhEyLGyXZSZhUkA97YrJWfEhZKgOIMGIAQUQU22SIlQQT2hGzy62JmB6vAZsl8mWilt522J2BYOLSvAnpCeFJnaTBhAzSCu4SyPFISv1+rdiQoEKORKTfgzSEGtw7YrhdmIrKQ+j6cCvQ2w4H3Mg2dSMtOxcl+dmo7U3F3p7JmF9agFWfuwUWy+iPeBhAMIAgIooLfknE9PSXiXYFZnX0uL1ayRHpjpgmn1vzJaSttt1ZPfSoo39Gh+RMuNOn6O6EtNxOrZXeEU2f9I4o/hy8yXlX/Do+rwe1h3ei9fhetHe7kI/ALkhtex+amy/ihT2nYSSkacPFr33ta6P6nhhAMIAgIorbMlGp6pBJoi3BMlHpiGk1R3aSqGHA2tOkzaCk5bWtp3FIiWhgd6IYnpRiLftMajky0DtCdiJkR+KKyZlSNeJqwdHX1qOu6gQy05NRaLqI80Y21u5tx7H9O/Q5mzZtGlUQwQCCAQQRUdyT/hLBJlYylrzTFUjEtEoiZkJkjzpMXldgd0ICCue5gSmaQdIoSgZiWTydOlnTZ0/V3IjerOsu23/C5/fh9tu/jIl99fjSdAcmFBZhQkYy1v3zIA6eatRdl6KiIlRXV4/4OGNcTuMkIiIKJzm6yEiy61WUGZgkKrkTsjMhFQ8dHR6YzSYk2awaUISzqsOwJsKdNVMv3Z1wNWsgoVd3PcxeFwyzDX6TBY7WY5Jdoa23ZZJo+7TVwx5rHK6sRHNTM5phxZn3+3Br/UnMLkzGsYbA0YnsB9TW1mrn52DDxkhiAEFERHE1STQ/PXHgqKO9O9DESkpF/Yah/SYkmAhre22TSZtAydWTt1BzJ2yddUjolICiRpMtpcW2rasOqV11WrXRmz1Hx5YHumJO1M9x8eLFgU/p9pnwem0C3nSm
```

Comment:
This seems to be producing a numpy `DeprecationWarning`, can  you update this to avoid this?